src/app/home/layouts/income-form/income-form.component.ts

```diff
@@ -13,58 +15,44 @@ import {basicAnimation} from '../../../shared/animation';
 export class IncomeFormComponent implements OnInit {
 
   incomeNumber = 1;
-
-  daysOfMonth = Array(31).fill(0).map((_, i) => i + 1);
-  lastDayOfMonth = this.daysOfMonth[this.daysOfMonth.length - 1];
-
-  activeDay: number;
-  previous: number;
-  next: number;
-
-  isSwipingUp = false;
-  isSwipingDown = true;
+  incomeDays: IncomeDaysDto[] = [];
 
   constructor(private initialSettingsService: InitialSettingsService,
               private router: Router) {
   }
 
-  ngOnInit(): void {
-    this.activeDay = 10;
+  ngOnInit() {
+    this.addIncomeDay();
     this.sendCurrentStepInfo();
-    this.setPreviousAndNextValue();
   }
 
   sendCurrentStepInfo() {
     this.initialSettingsService.sendCurrentStepInfo(2);
   }
 
-  toggleActiveOnSwipeUp() {
-    this.isSwipingUp = true;
-    this.isSwipingDown = false;
-
-    this.activeDay = this.activeDay < this.lastDayOfMonth ? this.activeDay + 1 : 1;
-    this.setPreviousAndNextValue();
-  }
-
-  toggleActiveOnSwipeDown() {
-    this.isSwipingDown = true;
-    this.isSwipingUp = false;
-
-    this.activeDay = this.activeDay > 1 ? this.activeDay - 1 : this.lastDayOfMonth;
-    this.setPreviousAndNextValue();
-  }
-
-  setPreviousAndNextValue() {
-    this.previous = this.activeDay !== 1 ? this.activeDay - 1 : this.lastDayOfMonth;
-    this.next = this.activeDay !== this.lastDayOfMonth ? this.activeDay + 1 : 1;
+  addIncomeDay() {
+    this.incomeDays.push({
+      active: 10,
+      previous: 9,
+      next: 11,
+      isSwipingUp: false,
+      isSwipingDown: true
+    });
   }
 
   addIncome() {
     this.incomeNumber += 1;
+    this.addIncomeDay();
   }
 
-  saveIncomes(form) {
-    console.log(form, this.activeDay);
+  saveIncomes(form: NgForm) {
+    const incomes = Object.values(form.form.value).map((element: IncomeFormDto, i) => {
+      return {
```

Comment:
There is no need for return here. You can just wrap the object in '( )' and place it next to arrow.